src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -412,6 +412,16 @@ ov::genai::ModelConfigDesc get_modeldesc_from_json(const std::filesystem::path&
     return desc;
 }
 
+std::string model_desc_to_string(const ov::genai::static_llm::ModelConfigDesc& model_desc) {
```

Comment:
We should prevent spreading this stub...

We should avoid having `ModelConfigDesc` at all as it tmp solution to solve two problems:
1. Specify `seq_len` dim for KV-cache
2. Figure out if need to enable `V` tensors transpose

I'd rather prefer having `SEQ_LEN_DIM` and `OPTIMIZE_V_TENSORS` than passing such config as `string`.